Minesweeper.java

```diff
@@ -62,13 +71,28 @@ public boolean equals(Object obj) {
         public int hashCode() {
             return Objects.hash(row, col);
         }
+
+        @Override
+        protected void paintComponent(Graphics g) {
+            super.paintComponent(g);
+
+            // Customize the appearance of the button to resemble a Minesweeper tile
+            if (isRevealed) {
+                setBackground(Color.GRAY);
+                setBorder(BorderFactory.createLoweredBevelBorder());
+            } 
+            else {
+                setBackground(Color.LIGHT_GRAY);
+                setBorder(BorderFactory.createRaisedBevelBorder());
+            }
+        }
     }
   
     private class Game extends JFrame{
-        private int gridSize;
+        private static int gridSize;
         private int cellSize;
         private int bombAmount;
-        private Cell[][] cells;
+        private static Cell[][] cells;
```

Comment:
this change isn't needed if we use `this` instead of `Game`, which we can if we get rid of the `getNeighboringBombs` method from `Cell`.
```suggestion
        private Cell[][] cells;
```